util/unifyfs/src/unifyfs-rm.c

```diff
@@ -539,6 +657,45 @@ static size_t construct_server_argv(unifyfs_args_t* args,
     return argc;
 }
 
+/**
+ * @brief Default data stage launch routine
```

Comment:
"Default data stage launch routine" doesn't really tell me what this function is doing.  What about, "Set default values for args[] and stage_argv[]"?

Reply:
This was just copied from the server_argv function.  I've changed them both to clarify and differentiate.